### PR TITLE
Feature/pusher frontend

### PR DESCRIPTION
**Pull request by goldquest**
This provides the basics for adding a notification plugin and implements it with the pusher library.

### CODE REVIEW

File: modules/liveblog_pusher/js/liveblog_pusher.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+(function(Drupal, drupalSettings) {
+  Drupal.behaviors.liveblogPusher = {
+    attach: function(context) {
+      Drupal.behaviors.liveblogStream.getContainer(context)
+        .once('liveblog-pusher-initialised')
+        .each(function(i, element) {
+          // Enable pusher logging - don't include this in production
+          Pusher.logToConsole = true;
```
Comment (milkovsky) [inline modules/liveblog_pusher/js/liveblog_pusher.js]: We can make a setting in the Pusher Plugin on the Drupal side in future for this. E.g. "Debugging mode".

File: modules/liveblog_pusher/liveblog_pusher.module [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?php
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\node\Entity\Node;
+
+/**
+ * Implements hook_ENTITY_TYPE_view().
+ */
+function liveblog_pusher_node_view(array &$build, Node $node, EntityViewDisplayInterface $display, $view_mode) {
+  if ($display->getComponent('liveblog_posts')) {
```
Comment (milkovsky) [inline modules/liveblog_pusher/liveblog_pusher.module]: we should check here if the current active plugin is pusher.
`if (Drupal::config('liveblog.notification_channel')->get('plugin')) == 'pusher'`

File: modules/liveblog_pusher/js/liveblog_pusher.js (hunk 1 above)
Comment (goldquest) [inline modules/liveblog_pusher/js/liveblog_pusher.js]: Yes, I will add a comment for that

File: modules/liveblog_pusher/liveblog_pusher.module (hunk 2 above)
Comment (goldquest) [inline modules/liveblog_pusher/liveblog_pusher.module]: This can also maybe be abstracted to the liveblog module. What do you think?

**Pull request merged**